testbed/tests/scenarios.go

```diff
@@ -88,9 +107,10 @@ processors:
 extensions:
   pprof:
     save_to_file: %v/cpu.prof
+  %s
 
 service:
-  extensions: [pprof]
+  extensions: [pprof, %s]
```

Comment:
Does this work correctly if `extensionsList` is empty?

Reply:
Yes, trailing commas are allowed in yaml lists and result in a list of only `pprof` and nothing else.